run_rayci.sh

```diff
@@ -16,4 +16,5 @@ export GOPRIVATE="github.com/ray-project/rayci"
 
 echo "--- Run rayci"
 
+export $(python ci/pipeline/determine_tests_to_run.py)
```

Comment:
no, not here..

I intentionally made this file simple, to just download run the rayci binary, without extra dependencies.

- the launch of the code needs to go into `rayci` binary.
- the entrypoint of the `determine_tests_to_run.py` needs to be in the config.
- and the output needs to be programmatically added into the env map rather than reading from system env.

Reply:
in short.. you need to write that in go, and add unit tests..

Reply:
finally, time to learn some go ;)